app/notifications/callbacks.py

```diff
@@ -22,6 +22,7 @@ def create_delivery_status_callback_data(notification, service_callback_api):
         "notification_client_reference": notification.client_reference,
         "notification_to": notification.to,
         "notification_status": notification.status,
+        "notification_status_description": notification.formatted_status,
```

Comment:
I didn't realize that the data returned for the callback is not the same as the data returned in the API. Good to know.